lib/manager/travis/extract.ts

```diff
@@ -22,6 +22,41 @@ export function extractPackageFile(content: string): PackageFile | null {
       currentValue: currentValue.toString(),
     }));
   }
+
+  // Handle the matrix syntax
+  let matrix_include: any;
```

Comment:
No `any`! Create and interface and add to `./types.ts`. Please also fix the `any` type of `doc`.